pkg/pipeline-selector/pipeline_selector_test.go

```diff
@@ -450,7 +450,32 @@ func TestGetPipelineSelectionParametersForComponent(t *testing.T) {
                 components:
                   - name: outerloop-deploy
                     kubernetes:
-                        uri: outerloop-deploy.yaml				  
```

Comment:
What is the reason behind this change?

Reply:
the devfile validation is more strict now and trying to resolve the file which does not exist.